### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse_ Azure Device Log in_ Keyboard: User has to press “Ctrl  + Tab” twice to come out of the table.

### ISSUE THREAD

**Issue by Accessibility56** [microsoft/azure-tools-for-java#3351]
[Check out Accessibility Insights!](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx) - Identify accessibility bugs before check-in and make bug fixing faster and easier

**Environment Details:**
Download and Install IntelliJ IDEA from https://docs.microsoft.com/en-us/java/azure/intellij/azure-toolkit-for-intellij-create-hello-world-web-app?view=azure-java-stable
IntelliJ IDEA 2019.1.3
Build #IU-191.7479.19
**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
**MAS Violated:** 2.4.3

**Repro Steps:**
1) Open IntelliJ IDEA application
2) Press 'Ctrl+ Alt+ S' to open application settings/preferences popup
3) Tab till 'Plugins' tab and press enter
4) Search for 'Azure Toolkit for IntelliJ in the Marketplace' and press enter.
5) Tab till 'Install' button and press enter.
6) Restart IntelliJ IDEA application.
7)Tab till “Azure Sign In” button and press enter.
8)Select “Service Principal” radio button and tab to “New” button and press enter. 	.
8) Tab till 'Copy and Open' button and press enter
9) Paste the code in the browser and press enter on next.
10)Tab till “Start” button in Create Authentication files pop up and press enter.
11)Verify focus order after table.

**Actual Result:**
User has to press “Ctrl  + Tab” twice to come out of the table.

**Expected Result:**
User has to press “Ctrl  + Tab” only once to come out of the table.

**User Impact:** 
Keyboard users get in understanding where the focus is if focus loss is observed after pop up.

**MAS Reference:** 
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}

[Bug_3351_Keyboard issue.pptx](https://github.com/microsoft/azure-tools-for-java/files/3451358/Bug_3351_Keyboard.issue.pptx)

**Comment by Flanker32** [comment]
@Accessibility56 This issue has been fixed in #3527, you may find the build [here](https://vscjavaci.cloudapp.net/job/Azure_Toolkits_for_Java/job/AzureToolsManager_develop/1247/Azure/), could you please help us review it? Thanks!

**Comment by Accessibility56** [comment]
Issue is fixed in the above build. Hence closing this bug.
[Uploading Bug_3351_Fixed.pptx…]()

**Comment by Accessibility56** [comment]
[Bug_3351_Fixed.pptx](https://github.com/microsoft/azure-tools-for-java/files/3620825/Bug_3351_Fixed.pptx)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749798
